xml/System.Diagnostics/ActivityCreationOptions`1.xml

```diff
@@ -38,8 +40,8 @@
         <ReturnType>System.Diagnostics.ActivityKind</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the activity kind associated with the activity.</summary>
```

Comment:
>Gets the activity kind associated with the activity [](start = 17, length = 51)

Get the ActivityKind value which the Activity will be created with.

Reply:
```suggestion
        <summary>Gets the ActivityKind value which the Activity will be created with.</summary>
```